brave/src/main/java/brave/handler/FinishedSpanHandler.java

```diff
@@ -64,9 +65,60 @@
    */
   public abstract boolean handle(TraceContext context, MutableSpan span);
 
+  /**
+   * Normally, {@link #handle(TraceContext, MutableSpan)} is only called upon explicit termination
+   * of a span: {@link Span#finish()}, {@link Span#finish(long)} or {@link Span#flush()}. When this
+   * method returns true, the callback will also receive data orphaned due to spans being never
+   * terminated or data added after termination. It is important to understand this, especially if
+   * your handler is performing work like redaction. This sort of work needs to happen on all data,
+   * not just the success paths.
+   *
+   * <p><h3>What is an orphaned span?</h3>
+   * Brave adds an {@link Span#annotate(String) annotation} "brave.flush" when data remains
+   * associated with a span when it is garbage collected. This is almost always a bug. For example,
+   * calling {@link Span#tag(String, String)} after calling {@link Span#finish()}, or calling {@link
+   * Tracer#nextSpan()} yet never using the result. To track down bugs like this, set the logger
+   * {@link PendingSpans} to FINE level.
+   *
+   * <p><h3>Why handle orphans?</h3>
+   * Use cases for handling orphans include redaction, trimming the "brave.flush" annotation,
+   * logging a different way than default, or incrementing bug counters. For example, you could use
+   * the same SSN cleaner here as you do on the success path.
+   *
+   * <p><h3>What shouldn't handle orphans?</h3>
+   * As this is related to bugs, no assumptions can be made about span count etc. For example, one
+   * span context can result in many calls to this handler, unrelated to the actual operation
+   * performed. Handlers that redact or clean data work for normal spans and orphans. However,
+   * aggregation handlers, such as dependency linkers or success/fail counters, can create problems
+   * if used against orphaned data.
+   *
+   * <p><h2>Implementation</h2>
+   * By default, this method returns false, suggesting the implementation is not designed to also
+   * process orphans. Return true to indicate otherwise. Whichever choice should be constant. In
+   * other words do not sometimes return false and other times true, as the value is only read
+   * once.
+   *
+   * <p><h3>Considerations for implementing {@code handle}</h3>
+   * When this method returns true, the {@link #handle(TraceContext, MutableSpan) handle method} is
+   * both invoked for normal spans and also orphaned ones. The following apply when handling
+   * orphans:
+   *
+   * <p>The {@link TraceContext} parameter contains minimal information, including lookup ids
+   * (traceId, spanId and localRootId) and sampling status. {@link TraceContext#extra() "extra"}
+   * will be empty.
+   *
+   * <p>The {@link MutableSpan} parameter {@link MutableSpan#containsAnnotation(String) includes
+   * the annotation} "brave.flush", and whatever state was orphaned (ex a tag).
```

Comment:
I might not be familiar enough with the terms, but what does it mean for a tag to be orphaned? Aren't spans the orphaned ones?

Reply:
terminology problem maybe you can help solve.

it is orphaned data associated with a span id. doesn't matter what kind, whether it is a single tag, a late name change, a redundant finish timestamp (though we guard against that)

so..
orphaned data
orphaned span

meh maybe better to be close as possible to right and change consistently to "orphaned span" at least we can defined it once. sgtm!